melodic/distribution.yaml

```diff
@@ -2069,6 +2069,23 @@ repositories:
       url: https://github.com/OTL/cv_camera.git
       version: master
     status: maintained
+  cyglidar_d1:
+    doc:
+      type: git
+      url: https://github.com/CygLiDAR-ROS/cyglidar_d1.git
+      version: v0.1.2
```

Comment:
This version is still confusing, in my opinion.  You are on branch v0.1.2, but the release version below is 0.0.2.  I won't block on this, but I still highly suggest that you make this a named branch on the source repository (`main`, or something like that), and use that instead.